docs/guide/tools/gh-issue-template.qmd

```diff
@@ -0,0 +1,32 @@
+---
+tite: "GitHub issue template"
+sidebar: guide
+---
+
+## GitHub Issue templates
+
+If you have your project in GitHub, you can create issue templates or forms. 
+These allow you ask relevant information to contributors wanting to open a new issue to your repository. 
+
+You can have a template for each type of issue. For example, it can be a bug report, 
+a request for a new feature, or a problem in the documentation. You could also ask for more 
```

Comment:
```suggestion
a request for a new feature, or a suggestion to improve  project's documentation. You could also ask for more 
```
Let's make it more general as it could also be adding more content to the existing project documentation.

Reply:
Done, thanks.